docs/source/release_notes.rst

```diff
@@ -2,12 +2,15 @@ Release Notes
 -------------
 **Future Releases**
     * Enhancements
+        * *First PyPi Release* :pr:`8`
```

Comment:
Why is this being added here? Releases shouldn't be added to the `release_docs`, I'd recommend following instructions similar to [these](https://github.com/alteryx/woodwork/blob/main/release.md)

Reply:
resolved